apps/communications/contacts/elements/settings.html

```diff
@@ -34,28 +34,6 @@ <h1 data-l10n-id="settings"></h1>
               </button>
             </li>
           </ul>
-          <gaia-subheader id="fb-header" skin="organic">
-            <span data-l10n-id="facebook">Facebook</span>
-          </gaia-subheader>
-          <ul id="settingsFb" role="presentation" data-type="list" data-state="logged-out">
```

Comment:
same here we need to remove from:

test/unit/mock_contacts_index.html

And in file: test/unit/views/settings_test.js
we have fbImportOption = document.querySelector('#settingsFb'); that's working cause we didn't remove from the mock html the dom.

Perhaps now it's not the moment to migrate to proper getting the html from the original instead of keeping those horrible mocks, but will be a task to consider in the future.